### ISSUE TITLE
[Analytics] [Campaign Landing Pages] Custom GA events removal & web components upgrades

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#18507]
## Blocked
Blocked by this DST defect: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2817

We cannot convert this to a `va-link` until that accessibility defect is addressed.

## Description
The va-link web component's analytics now include link destination, and va-link-action has been released to the component library. We can now use baked-in analytics for Campaign Landing Pages download links.

### User story
**AS A PO/PM** managing Sitewide products & features
**I WANT** to (where appropriate) adopt the va-link or va-link-action component, its baked-in analytics, and remove the custom GA events
**SO THAT** extraneous code is removed and only the events needed will be triggered.

### Engineering notes
Content-build template: https://github.com/department-of-veterans-affairs/content-build/blob/7c123600579e02589bbfc0b17408492e6adc363c/src/site/layouts/campaign_landing_page.drupal.liquid#L299

## Acceptance criteria
- [ ] Element: "Download ___ PDF" links
  - [ ] upgrade to web component
  - [ ] enable web component analytics
- [ ] a11y review

**Comment by sky3an1co** [comment]
Blocked by department-of-veterans-affairs/vets-design-system-documentation/issues/3537
